Shared/sdk/SharedUtil.Crypto.h

```diff
@@ -32,6 +33,60 @@ namespace SharedUtil
         return result;
     }
 
+    inline std::pair<std::string, std::string> GenerateRsaKeyPair(const int size)
```

Comment:
Take `unsigned int`, because `GenerateRandomWithKeySize` expects an unsigned int as well.
```suggestion
    // Returns a pair of private and public keys, both of the specified size.
    inline std::pair<std::string, std::string> GenerateRsaKeyPair(const unsigned int size)
```